vendor/bat-native-ads/src/bat/ads/internal/account/utility/redeem_unblinded_token/create_confirmation_util.cc

```diff
@@ -46,7 +46,7 @@ std::string CreateConfirmationRequestDTO(const ConfirmationInfo& confirmation) {
     confirmation_request_dto.Set("publicKey", *value);
   }
 
-  const absl::optional<base::Value> user_data =
+  absl::optional<base::Value> user_data =
```

Comment:
@cdesouza-chromium `const`

Reply:
@tmancey `const` is causing a build error here:

<pre>
../../brave/vendor/bat-native-ads/src/bat/ads/internal/account/utility/redeem_unblinded_token/create_confirmation_util.cc(52,36): error: call to deleted constructor of 'base::Value::Dict'
    confirmation_request_dto.Merge(std::move(user_data->GetDict()));
                                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../..\base/values.h(377,5): note: 'Dict' has been explicitly marked deleted here
    Dict(const Dict&) = delete;
    ^
../..\base/values.h(444,21): note: passing argument to parameter 'dict' here
    void Merge(Dict dict);
                    ^
1 error generated.
</pre>

Reply:
Ah, yeah now I remember why I had removed the constness in this case, as it helps to avoid cloning the data from the dictionary, which was not possible in the previous API.

Reply:
I will check other ad usage and fix it where needed in a future PR. Thanks